clang/bindings/python/clang/cindex.py

```diff
@@ -2052,6 +2052,19 @@ def referenced(self):
 
         return self._referenced
 
+    @property
+    def specialized(self):
+        """
+        For a cursor that is instantiated from a template, returns a cursor
+        representing the template that it was instantiated from.
+        """
+        if not hasattr(self, "_specialized"):
+            self._specialized = Cursor.from_result(
+                conf.lib.clang_getSpecializedCursorTemplate(self), self
+            )
+
+        return self._specialized
+
```

Comment:
This change seems unrelated to the rest of this PR, and should be in a separate one. It is also missing tests.

Reply:
removed this and will put into a different PR (with tests)